frontend/src/lib/components/ExportButton/exportsLogic.ts

```diff
@@ -146,6 +147,14 @@ export const exportsLogic = kea<exportsLogicType>([
                 },
             }
 
+            actions.startExport(exportData)
+        },
+        startHeatmapExport: async ({ export_context }) => {
```

Comment:
i think ultimately there's something value-add if not chargeable in saving images without heatmaps and doing diffing between them so we can tell people what changed